firmware/Sources/HAL/HAL_SYS/hal_sys.c

```diff
@@ -121,9 +121,9 @@ HAL_SYS_result_e HAL_SysInit(void){
 #endif
 
 #ifdef EPC_CONF_TMR_ENABLED
-			res = HAL_TmrInit(HAL_TMR_CLOCK_RT);
-			res |= HAL_TmrInit(HAL_TMR_CLOCK_PWR_MEAS);
-			if (res != HAL_TMR_RESULT_SUCCESS){
+			HAL_TMR_result_e tmr_res = HAL_TmrInit(HAL_TMR_CLOCK_RT);
```

Comment:
STM HAL returns 0 when SUCCESS?

Reply:
Yes, and perform |= operation to detect a error.